content/general-docs/introduction.adoc

```diff
@@ -93,7 +93,7 @@ The defined look-up tables include:
 * xref:material/material-emp-schema.adoc[Electromagnetic properties]
 * xref:material/material-optical-schema.adoc[Optical properties]
 * xref:material/material-brdf-schema.adoc[Bidirectional reflectance distribution functions (BRDFs)]
-* xref:material/material-reflectance-schema.adoc[Reflectance table]
+//* xref:material/material-reflectance-schema.adoc[Reflectance table]
```

Comment:
I see the reflectance schema is missing from the navigation bar. So it has to be the other way around. This does not have to be commented out, but the navigation bar has to be fixed.